base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -2668,24 +2607,14 @@ where
         {
             if output.verify_mask(&self.resources.factories.range_proof, &spending_key, amount.as_u64())? {
                 let spending_key_id = self.resources.key_manager.import_key(spending_key).await?;
-                let rewound_output = WalletOutput::new_with_rangeproof(
-                    output.version,
+
+                let rewound_output = WalletOutput::new_from_transaction_output(
```

Comment:
In 4x places - we do not have a notion of a `rewound_output` anymore.
```suggestion
                let recovered_output = WalletOutput::new_from_transaction_output(
```